package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "safe-redirect-manager",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
This needs to be updated in package-lock.json too, in two places:

https://github.com/10up/safe-redirect-manager/blob/620210a00745f396c3a13ef71841a38068520991/package-lock.json#L3

https://github.com/10up/safe-redirect-manager/blob/620210a00745f396c3a13ef71841a38068520991/package-lock.json#L9